CMS/urls.py

```diff
@@ -36,7 +36,7 @@
     url(r'', include(wagtail_urls)),
 ]
 
-if not settings.READ_ONLY:
+if True or not settings.READ_ONLY:
```

Comment:
are you sure this change is correct?

Reply:
It was required for the local dev environment. Will remove.